backend/dissemination/models.py

```diff
@@ -266,16 +242,10 @@ class FederalAward(models.Model):
     #     help_text=docs.questioned_costs_FederalAward,
     # )
 
-    # SK - Note: Need to add the following:
-    # audit_report_type
-    # elecauditsid
-    # federal_agency_prefix
+    # TODO: Do we need to add any of the following?:
```

Comment:
If the question is "will we need is_guaranteed, is_passed, or subrecipient_amount?," the answer will be yes.